import logging

def test_example_1():
    logging.error('some test error')
